exec/cnex/number.h

```diff
@@ -34,8 +34,16 @@
 
 typedef BID_UINT128 Number;
 
-void number_toString(Number x, char *buf, size_t len);
+typedef enum enNumberFormat {
+    nfFull,
+    nfScientific,
+} NumberFormat;
+
+NumberFormat NumberDisplayMode;
```

Comment:
Needs to be `extern` when declared in a header file.

Reply:
Fixed.  I guess I had them backwards.